manifests/s/ShiningLight/OpenSSL/3.0.8/ShiningLight.OpenSSL.installer.yaml

```
@@ -1,50 +1,49 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
-
-PackageIdentifier: ShiningLight.OpenSSL
-PackageVersion: 3.0.7
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-UpgradeBehavior: install
-Commands:
-- openssl
-Installers:
-- InstallerLocale: en-US
-  Architecture: x64
-  InstallerType: wix
-  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-3_0_7.msi
-  InstallerSha256: 9EE54A28560B9845759755D39E42CFAC475B65F9A2CCD63F394279CAF04DBAA5
-  ProductCode: '{43F3F4EA-4B60-42BB-8E06-54E3DD2329D0}'
-- InstallerLocale: en-US
-  Architecture: x86
-  InstallerType: wix
-  InstallerUrl: https://slproweb.com/download/Win32OpenSSL-3_0_7.msi
-  InstallerSha256: AEEEAC29F496F153853C2BB55390A584191FEADAB1575F9B156CC4A1122A6BB3
-  ProductCode: '{92E5CC00-89D8-45F3-B5FC-3618E7B50FB0}'
-- Architecture: x64
-  InstallerType: inno
-  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-3_0_7.exe
-  InstallerSha256: 8E865F43585D7EEE5BDF387D259DE8E26F1D28798657C51033C176CEC02CAB5A
-  ProductCode: OpenSSL (64-bit)_is1
-  AppsAndFeaturesEntries:
-  - DisplayName: OpenSSL 3.0.5 (64-bit)
-    Publisher: OpenSSL Win64 Installer Team
-    ProductCode: OpenSSL (64-bit)_is1
-- Architecture: x86
-  InstallerType: inno
-  InstallerUrl: https://slproweb.com/download/Win32OpenSSL-3_0_7.exe
-  InstallerSha256: BC97AA3D6BEE0A2B4320148D79D617603213260C5C48E0C299C1A373807862F5
-  ProductCode: OpenSSL (32-bit)_is1
-  AppsAndFeaturesEntries:
-  - DisplayName: OpenSSL 3.0.5 (32-bit)
-    Publisher: OpenSSL Win32 Installer Team
-    ProductCode: OpenSSL (32-bit)_is1
-ManifestType: installer
-ManifestVersion: 1.2.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.0.8
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- openssl
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-3_0_8.msi
+  InstallerSha256: 7AA094E9A4E079E5E3D9E5829BB0EEB187F99919D0119759B88331E03DA3BB77
+  ProductCode: '{43F3F4EA-4B60-42BB-8E06-54E3DD2329D0}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL-3_0_8.msi
+  InstallerSha256: 545111206C0162628180C48A74EF765DB34B04AC796EE3E26C924C502E5A7E8A
+  ProductCode: '{92E5CC00-89D8-45F3-B5FC-3618E7B50FB0}'
```

```suggestion
  ProductCode: '{F720463B-451A-4E56-B2BB-35FC651B6112}'
```
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{F720463B-451A-4E56-B2BB-35FC651B6112}`